Airports/EGNO/SMR/Labels.txt

```diff
@@ -0,0 +1,13 @@
+EGNO Holding Points              S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```

Comment:
This file is actually the Geo

The "EGNO Holding Points" should become "EGNO Warton"